include/types.h

```diff
@@ -24,13 +24,37 @@
 #define SCREEN_CENTER_X (SCREEN_WIDTH  / 2)
 #define SCREEN_CENTER_Y (SCREEN_HEIGHT / 2)
 
+typedef union {
+    struct PACKED {
```

Comment:
i don't think the packed attribute is doing much here

Reply:
Reverted